5.5/Dockerfile.rhel7

```diff
@@ -1,4 +1,4 @@
-FROM openshift/base-rhel7
+FROM registry.access.redhat.com/rhscl/s2i-base-rhel7
```

Comment:
I believe changing to `rhscl/s2i-base-rhel7` should be enough, because `registry.access.redhat.com` is configured by default in RHEL build of docker. Also, we don't want to hardcode the registry url in the Dockerfile, because it would prevent anybody to use own base image locally or from different registry.